guides/advanced_guides/search_parameters.md

```diff
@@ -294,6 +298,8 @@ If the `facets` parameter has been set, the returned results will contain two ad
   Returns `true` if this count is **exhaustive**.
   Otherwise, returns `false` if this count is **approximative**.
 
+[Learn more about the facet filters](/guides/advanced_guides/faceted_search.md#_2-the-facets-distribution)
```

Comment:
```suggestion
[Learn more about the facet filters](/guides/advanced_guides/faceted_search.md#_2-the-facets-distribution).
```

Reply:
Why the link goes to facets distributions and talks about facet filters?

Reply:
It's still wip, I changed it.

Reply:
Sorry, I've just seen the "don't review". My bad. Could you pass it as a draft instead? So that there is no confusion ^^

Reply:
Yes, I changed it!